api/v1/authenticator_types.go

```diff
@@ -20,6 +20,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type IfState string
+
+var (
+	IfStateUninitialized IfState = "Uninitialized"
+	IfStateDisabled      IfState = "Disabled"
+	IfStateCountryUpdate IfState = "CountryUpdate"
+	IfStateAcs           IfState = "ACS"
+	IfStateHtScan        IfState = "HT Scan"
+	IfStateDfs           IfState = "DFS"
+	IfStateEnabled       IfState = "Enabled"
+	IfStateUnknown       IfState = "Unknown"
```

Comment:
Are all of these valid for 802.1x, or are some only valid for WiFi authentication (which is the other thing hostapd does)?  Can we get a one-liner about what each one means?

Reply:
sure, will do it with another commit.